include/libtorrent/torrent.hpp

```diff
@@ -1300,9 +1297,9 @@ namespace libtorrent
 		// m_num_verified = m_verified.count()
 		std::uint32_t m_num_verified = 0;
 
-		// this timestamp is kept in session-time, to
-		// make it fit in 16 bits
-		std::uint16_t m_last_saved_resume;
+		// TODO: waste of memomry because has a max session time offset of
+		// optimistic_disk_retry
```

Comment:
That means you don't want to change it in the next optimization step? My idea was to put a todo on each position where I did at least double the memory usage.